test/e2e/framework.go

```diff
@@ -1544,7 +1544,7 @@ func (data *TestData) DeletePodAndWait(timeout time.Duration, name string, ns st
 		return false, nil
 	})
 	if err == wait.ErrWaitTimeout {
-		return fmt.Errorf("Pod '%s' still visible to client after %v", name, timeout)
+		return fmt.Errorf("pod '%s' still visible to client after %v", name, timeout)
```

Comment:
not necessary, I think we keep it capitalized given that we always capitalize "Pod"?